### PR TITLE
Moved clientConfig resolver into falcon-front-kit

### PR DESCRIPTION
**Pull request by tutok**
### Please check if the PR fulfills these requirements ###

- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**
resolves https://github.com/deity-io/falcon/issues/577

**Does this PR introduce a breaking change? (What changes might users need to make in their application due to this PR?)**
YES, dev needs to register `clientConfig` resolver if he wants to query it

**Other information:**
(please forgive me for Czech mistake in branch name)

### CODE REVIEW

File: packages/falcon-front-kit/src/Apollo/apolloStateToObject.ts [hunk 1]
```diff
@@ -4,8 +4,9 @@
  * @param {string} key Apollo state key
  * @returns {object} Expanded object
  */
-export const apolloStateToObject = (state, key) => {
+export function apolloStateToObject(state: Object, key: string): Object {
```
Comment (Borales) [inline packages/falcon-front-kit/src/Apollo/apolloStateToObject.ts]:
```suggestion
export function apolloStateToObject(state: object, key: string): Object {
```

shouldn't it be `object`?

File: examples/shop-with-blog/client/src/clientState.js [hunk 2]
```diff
@@ -59,6 +61,7 @@ export default {
 
   resolvers: {
     Query: {
+      clientConfig: getClientConfigResolver,
```
Comment (Borales) [inline examples/shop-with-blog/client/src/clientState.js]: @tutok would it make sense adding this to `UPGRADE` notes?

Comment (mateuszwozniak) [reply examples/shop-with-blog/client/src/clientState.js]: Whole application needs to be upgraded so I'm not sure if it makes sense...

Comment (Borales) [reply examples/shop-with-blog/client/src/clientState.js]: @mateuszwozniak solid argument 😄 closing this one

File: packages/falcon-front-kit/src/Apollo/apolloStateToObject.ts (hunk 1 above)
Comment (tutok) [reply packages/falcon-front-kit/src/Apollo/apolloStateToObject.ts]: done

**Pull request merged**